website/docs/docs/cloud/manage-access/auth0-migration.md

```diff
@@ -30,7 +30,7 @@ Once you have opted to begin the migration process, the following steps will var
 
 :::warning Login {slug}
 
-The login slug shouldn't have an underscore. If it does, you need to change it on the **Account Settings** page. After changing the slug, the admins must share it with dbt Cloud users, as the login URL will differ.
+The login slug shouldn't have an underscore. If it does, you need to change it before migrating on the **Account Settings** page or during the migration on the final screen before enabling (see screenshots for your respective setup). After changing the slug, the admins must share it with dbt Cloud users, as the login URL will differ.
```

Comment:
Is the action here to remove the underscore from all logins? Maybe this message can be condensed:
```suggestion
Make sure to remove underscores (if they exist) from login slugs: 
* before migrating on the **Account Settings** page, or 
* while migrating (but before enabling) as show in the Migrate authentication screenshots for your respective setup. 
After changing the slug, admins must share the new login URL with their dbt Cloud users.
```